examples/multi-project/README.md

```diff
@@ -18,7 +18,7 @@ jobs:
       vmImage: ubuntu-latest
 
     steps:
-      - task: TerraformInstaller@0
+      - task: TerraformInstaller@0  # this can be obtained by installing the microsoft terraform extension: https://marketplace.visualstudio.com/items?itemName=ms-devlabs.custom-terraform-tasks
```

Comment:
```suggestion
      - task: TerraformInstaller@0  # This can be obtained by installing the Microsoft Terraform extension: https://marketplace.visualstudio.com/items?itemName=ms-devlabs.custom-terraform-tasks
```